src/main/plugins/org.dita.pdf2/xsl/fo/loc.xsl

```diff
@@ -0,0 +1,231 @@
+      <fo:block>
+        <fo:table xsl:use-attribute-sets="releaseManagementTable">
+          <xsl:call-template name="selectAtts"/>
+          <xsl:call-template name="globalAtts"/>
+          <fo:table-body>
+            <fo:table-row xsl:use-attribute-sets="releaseManagementTable.bookRelease.row">
+              <fo:table-cell xsl:use-attribute-sets="releaseManagementTable.bookRelease.cell">
+                <fo:block xsl:use-attribute-sets="releaseManagementTable.bookRelease.content">
+                  <fo:block>
+                    <xsl:value-of select="$bookReleaseString"/>
+                    <xsl:value-of select="$colonSymbol"/>
+                    <xsl:value-of select="$i + 1"/>
+                  </fo:block>
+                  <fo:block>
+                    <xsl:value-of select="$bookReleaseDateString"/>
+                    <xsl:value-of select="$colonSymbol"/>
+                    <xsl:value-of select="$separator"/>
+                    <xsl:value-of select="format-date($timeRangeEndAsDate, $dateFormat)"/>
+                  </fo:block>
+                </fo:block>
+              </fo:table-cell>
+            </fo:table-row>
+            <!-- Iterate over all change-items -->
+            <xsl:for-each select="$changeset">
+              <xsl:variable name="j" select="position()"/>
+              <xsl:variable name="currentChangeItem" select="$changeset[$j]"/>
+              <xsl:variable name="currentChangeItemChangeCompleted" select="$changeset[$j]/*[contains (@class, ' relmgmt-d/change-completed ')]"/>
+              <xsl:if test="$timeRangeStart != ''
+                            and $currentChangeItemChangeCompleted != ''
+                            and $timeRangeEnd">
+                <xsl:variable name="currentChangeItemChangeCompletedAsDate" select="xs:date($currentChangeItemChangeCompleted)"/>
+                <xsl:if test="($timeRangeStartAsDate &lt;= $currentChangeItemChangeCompletedAsDate) and ($currentChangeItemChangeCompletedAsDate &lt;= $timeRangeEndAsDate)">
+                  <fo:table-row xsl:use-attribute-sets="releaseManagementTable.changeItem.row">
+                    <fo:table-cell xsl:use-attribute-sets="releaseManagementTable.changeItem.cell">
+                      <fo:block xsl:use-attribute-sets="releaseManagementTable.changeItem.content">
+                        <xsl:apply-templates mode="relmgmt-table" select="$currentChangeItem">
+                          <xsl:with-param name="change-item" select="$currentChangeItem"/>
+                        </xsl:apply-templates>
+                      </fo:block>
+                    </fo:table-cell>
+                  </fo:table-row>
+                </xsl:if>
+              </xsl:if>
+            </xsl:for-each>
+          </fo:table-body>
+        </fo:table>
+      </fo:block>
+    </xsl:for-each>
+  </xsl:template>
+
+  <xsl:template match="*[contains(@class, ' relmgmt-d/change-item ')]" mode="relmgmt-table">
+    <xsl:param name="change-item"/>

```

Comment:
`change-item` parameter is the same node as the current node, refactor to use current node